tests/s3/versioning/test_get_bucket_versioning.py

```diff
@@ -70,12 +72,16 @@ def teardown_method(self):
         if path_exists(self.test_dir_path):
             remove_dirs(self.test_dir_path)
         self.log.info("Cleanup test directory: %s", self.test_dir_path)
-        res = self.s3_test_obj.bucket_list()
-        pref_list = [
-            each_bucket for each_bucket in res[1] if each_bucket.startswith("ver-bkt")]
-        if pref_list:
-            res = self.s3_test_obj.delete_multiple_buckets(pref_list)
-            assert_utils.assert_true(res[0], res[1])
+        # TODO: DELETE Object with VersionId is WIP, uncomment once feature is available
```

Comment:
If you anticipate it late then remove this commented code.

Reply:
Anticipating the changes within current or next sprint, retaining the code as per our offline discussion on this matter.